### DISCUSSION TITLE
client.read_data / intersection_otherlane

### DISCUSSION THREAD

**Discussion by Miknow78** [carla-simulator/carla#5001] [Q&A]
Hi!
I'm trying to check when the car changes the lane.
I saw the function client.read_data() in stable version that gives the percentage of the car invading other lanes (intersection_otherlane attribute).
Something like this:  measurements, data_sensor = client.read_data()
It shows me an error: 'Client' object has no attribute 'read_data'. 

Does anyone know any replacement function?

**Comment by mjpablo23** [comment]
I am having trouble finding client.read_data also
